lib/Cpdf.php

```diff
@@ -4871,7 +4871,7 @@ protected function addImagePngAlpha($file, $x, $y, $w, $h, $byte)
             if ($imagick->getImageAlphaChannel() !== 0) {
                 $alpha_channel = $imagickClonable ? clone $imagick : $imagick->clone();
                 $alpha_channel->separateImageChannel(\Imagick::CHANNEL_ALPHA);
-                $alpha_channel->negateImage(true);
+                $alpha_channel->negateImage(true, \Imagick::CHANNEL_ALPHA);
```

Comment:
Is this the right channel to use? Based on my reading, the default channel changed from RGB to RGBA. So for this operation should we be using `\Imagick::CHANNEL_RED | \Imagick::CHANNEL_GREEN | \Imagick::CHANNEL_BLUE`?

Reply:
for ImageMagick 7 negate in principle not need
may be Imagick::getVersion() and check version?

Reply:
I'm open to optimizing based on version.